scripts/generate-wayland-protocols.sh

```diff
@@ -9,7 +9,7 @@ if [ "$GLGLFW_PATH" = "" ]; then
     exit 1
```

Comment:
Consider adding [`set -e`](https://explainshell.com/explain?cmd=set+-e) at the top. That would've helped catch the issue with `git clone` failing silently.

Reply:
Nice idea. Fixed that in the latest commit